### PR TITLE
AnastasiaTre-Board-1

### PR DESCRIPTION
**Pull request by Anastasia-Tre**

### CODE REVIEW

File: Quoridor.Model/BoardFactory.cs [hunk 1]
```diff
@@ -0,0 +1,27 @@
+﻿namespace Quoridor.Model {
+
+    public interface IBoardFactory {
```
Comment (VsIG-official) [inline Quoridor.Model/BoardFactory.cs:3]: Place this interface in another file

File: Quoridor.Model/Board.cs [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+        public void getPossibleWallsPlaces() {
+
+        }
+
+        Cell getCellById(int id) {
+            foreach (Cell elem in cells) {
+                if (elem.id == id) return elem;
+            }
+            return null;
```
Comment (VsIG-official) [inline Quoridor.Model/Board.cs:31]: Maybe there We should use NullObject pattern?

**Review by VsIG-official — APPROVED** [review]
Suggestions about style You can see [there](https://github.com/Sierra-KPI/Quoridor/pull/8#pullrequestreview-762950790).
Also We should think about NullObject Pattern

**Review by LovelyDrug — APPROVED** [review]

**Comment by VsIG-official** [comment]
Nice one

**Pull request merged**